controllers/rabbitmqcluster_controller.go

```diff
@@ -51,14 +51,16 @@ import (
 )
 
 var (
-	apiGVStr = rabbitmqv1beta1.GroupVersion.String()
+	apiGVStr       = rabbitmqv1beta1.GroupVersion.String()
+	NewPodExecutor = func() KubectlExecutor { return &podExecutor{} }
```

Comment:
I was hesitant to change the signature of the Reconciler in case this broke anything to do with the kubebuilder scaffolding. I'll give it a shot though!